Cappuccino.py

```diff
@@ -1,3 +1,8 @@
 # Solide, verbreitet, gut
 # ein echter Klassiker
-print("Cappuccino - 1,00€")
+
+def name_with_price(price: str):
```

Comment:
knhihi - types in python - alright

Reply:
There is no python besides python 3, this also ensures that :D